### DISCUSSION TITLE
CMI Rankup

### DISCUSSION THREAD

**Discussion by NyhmsQuest** [Zrips/CMI#6511] [Q&A]
Currently I have various requirements people need to meet in order to rankup, for example mine 64 diamond ore, but with 1.18 and the new deepslate variants I was wondering if it's possible to check both deepslate and normal diamond ore as the same requirement?
Same goes for tree logs, is it possible to combine all trees into the same Logs Cut requirement?

I'm using
BlocksMined:diamond_ore:64

**Comment by CloudeLecaw** [comment]
thats not possible. You find here what is possible
https://www.zrips.net/cmi/rankup-system/

Problem is you cant combine two typed of items. there is no combined item called wood or stone for example.
It might worth to work with stats here like blocks mined etc..
